Sources/RealDeviceMap/Controller/InstanceControllers/CircleInstanceController.swift

```diff
@@ -30,6 +31,10 @@ class CircleInstanceController: InstanceControllerProto {
     private var lock = Threading.Lock()
     private var lastLastCompletedTime: Date?
     private var lastCompletedTime: Date?
+    private var currentUuidIndexes: [String: Int]
+    private var currentUuidSeenTime: [String: Date]
+    public let useRwForRaid =
```

Comment:
this causes to the non default behaviour... I have not set this environment variable.... so `ProcessInfo.processInfo.environment["USE_RW_FOR_RAID"]`would cause `nil`.... so the output would be false.....
I think it's the same with quests....

Reply:
well he suggested that default would be to not use rw warning accounts for quests since we see now that's a problem.
to make it consistent i took the option to also not use rw warning accounts for raids as default (accounts are just so easy to get now, there's no reason really to not want this).